src/language/class-modifiers.md

```diff
@@ -194,15 +205,16 @@ class MockVehicle          // ERROR: Cannot be implemented
 
 ## `sealed`
 
-Use the `sealed` modifier to create a known, enumerable set of subtypes, so that
-a switch over those types can be statically ensured to be [_exhaustive_][exhaustive].
+To create a known, enumerable set of subtypes, so that a switch over those types
+can be statically ensured to be [_exhaustive_][exhaustive], use the `sealed` modifier.
 
 The `sealed` modifier prevents a class from being extended or implemented outside
-of the library where the class is defined. The compiler is aware of any possible
-direct subtypes because they can only exist in the same library.
+of the library where the class is defined. They also prevent construction,
+and are therefore implicitly [abstract](#abstract).
 
-This allows the compiler to alert you when a switch is not exhaustively handling
-all possible subtypes in its cases:
+The compiler is aware of any possible direct subtypes because they can only exist
+in the same library. This allows the compiler to alert you when a switch does not
+exhaustively handle all possible subtypes in its cases:
```

Comment:
**suggestion:** Not sure if this is better. I'm not sure if another construction could handle "exhaustively" better. Ignore if desired.
```suggestion
As possible direct subtypes can only exist in the same library,
their limited scope makes the compiler aware of them.
This limitation allows the compiler to alert you when a `switch` does not
exhaust all possible subtypes in its `case` statements:
```